src/clientversion.cpp

```diff
@@ -77,23 +77,19 @@ std::string FormatSubVersion(const std::string& name, int nClientVersion, const
     return ss.str();
 }
 
-std::string CopyrightHolders(const std::string& strPrefix)
+std::string CopyrightHolders(const std::string& strPrefix1, const std::string& strPrefix2)
```

Comment:
I think it would still make sense for our copyright string changes to be in a separate commit

Reply:
I agree.  I've created another commit as part of this PR that has all copyright changes for Vertcoin